src/Features/Features.php

```diff
@@ -389,4 +400,17 @@ public static function add_admin_body_classes( $admin_body_class = '' ) {
 		$admin_body_class = implode( ' ', array_unique( $classes ) );
 		return " $admin_body_class ";
 	}
+
+	/**
+	 * Alias moved features classes to make them backward compatible.
+	 * We've moved our features class to src/Internal/Admin as part of The Merge.
+	 * See https://wp.me/p90Yrv-2HY for details.
+	 *
+	 * @param array $aliases an array of class aliases.
+	 */
+	private function register_moved_class_alisases( array $aliases ) {
```

Comment:
typo in function name

Reply:
Fixed all three in https://github.com/woocommerce/woocommerce-admin/pull/8307/commits/042dac8bcebd26febbb89c9d14163a7e9ec59b82

Thank you for spotting them 🙇‍♂️